storage/innobase/srv/srv0srv.cc

```diff
@@ -1722,6 +1775,9 @@ static uint32_t srv_do_purge(ulint* n_total_purged)
 	static uint32_t	rseg_history_len = 0;
 	ulint		old_activity_count = srv_get_activity_count();
 	static ulint	n_threads = srv_n_purge_threads;
+	static lsn_t	adaptive_purge_threshold = 20;
+	static lsn_t	safety_net = 20;
+	static ulint	series[innodb_purge_threads_MAX + 1];
```

Comment:
Maybe there should be a `static` object that includes all this data, such as `purge_coordinator.rseg_history_len`. In that way, the data would be more accessible in a debugger.

Reply:
1. cleanup sounds good .. we can consider it but I am still trying to understand how it helps debugging. even the existing static variables are acccessible. (am I missing anything?)